modules/python3/processors/PythonExample.py

```diff
@@ -0,0 +1,34 @@
+# Name: PythonExample 
```

Comment:
File name is using camel case. Does it have to to match with the class name?

Reply:
As long as you have the name comment the filename can be anything.py.. if there is no name comment it uses the filename